### ISSUE TITLE
term-suppress-hard-newline (obsolete variable)

### ISSUE THREAD

**Issue by ghost** [tpapp/julia-repl#97]
Emacs 27.1
Julia 1.5.1
julia-repl (git clone)

Byte-compile julia-repl.el.

```
In julia-repl--setup-term:
julia-repl.el:281:51:Warning: ‘term-suppress-hard-newline’ is an obsolete
    variable (as of 27.1).
```

**Comment by tpapp** [comment]
Thanks, I will fix this once I merge #84.

**Comment by tpapp** [comment]
I am not sure why you closed this, reopened as a reminder.

**Comment by tpapp** [comment]
Will fix this once #102 is done, would prefer to have CI for a setup that works on all Emacs versions.

**Comment by tpapp** [comment]
Thanks for your patience, and sorry about the delay addressing this.

I am not sure why this is not caught by CI now, but it should be fixed in a minute.
